framework/src/play/Play.java

```diff
@@ -726,7 +727,7 @@ public static void loadModules(VirtualFile appRoot) {
         if (System.getenv("MODULES") != null) {
             // Modules path is prepended with a env property
             if (System.getenv("MODULES") != null && System.getenv("MODULES").trim().length() > 0) {
-                for (String m : System.getenv("MODULES").split(System.getProperty("os.name").startsWith("Windows") ? ";" : ":")) {
+                for (String m : System.getenv("MODULES").split(Os.isWindows() ? ";" : ":")) {
```

Comment:
Can't we use just `File.pathSeparator` instead of this tricky hack? 
According to javadoc,
> On UNIX systems, this character is <code>':'</code>; on Microsoft Windows systems it is <code>';'</code>.